integration/container_stats_test.go

```diff
@@ -306,7 +306,6 @@ func testStats(t *testing.T,
 	require.NotEmpty(t, s.GetAttributes().GetId())
 	require.NotEmpty(t, s.GetAttributes().GetMetadata())
 	require.NotEmpty(t, s.GetAttributes().GetAnnotations())
-	require.Equal(t, s.GetAttributes().GetLabels(), config.Labels)
```

Comment:
Why is this changed?

Reply:
@abhi I don't quite get it, sorry.

Reply:
The container config will have only the added labels. Not the labels added by Cri-containerd. for eg config.Labels will be [key:value] , but the metadata store will have the [key:value cri-containerd.kind:"sandbox"]. I was hoping to take care of these things in the stats test cleanup

Reply:
i was thinking we had to store the labels we set for containerd as well. I will make the correction in the labels code above.